### PR TITLE
feat: beautysh formatter for bash, sh, and zsh files

### PR DESCRIPTION
**Pull request by vladdoster**
Signed-off-by: Vladislav Doster <mvdoster@gmail.com>

### CODE REVIEW

**Comment by vladdoster** [comment]
@jose-elias-alvarez Ready for review

File: doc/BUILTINS.md [hunk 1]
```diff
@@ -1734,6 +1734,27 @@ local sources = { null_ls.builtins.formatting.bean_format }
...
+```lua
+local sources = { null_ls.builtins.formatting.beautysh }
+```
+
+#### Defaults
+
+- Filetypes: `{ "bash", "sh", "zsh" }`
+- Method: `formatting`
+- Command: `beautysh`
+- Args: `{ "--indent-size", 2, "$FILENAME" }`
```
Comment (jose-elias-alvarez) [inline doc/BUILTINS.md:1752]: Question: does it make sense to hardcode this as a default, or should we leave it up to the user to configure?

Comment (vladdoster) [reply doc/BUILTINS.md:1752]: Agreed. Will fix sometime today.

**Comment by jose-elias-alvarez** [comment]
Thanks!

**Pull request merged**